src/sentry/api/helpers/group_index/update.py

```diff
@@ -738,3 +715,45 @@ def update_groups(
         result["inbox"] = inbox
 
     return Response(result)
+
+
+def handle_is_public(
+    is_public: Any,
```

Comment:
Can this be typed as a boolean?

Reply:
type check doesn't succeed so we'd have to explicitly cast it